### ISSUE TITLE
[BUG] Intellisense not working in VS Code

### ISSUE THREAD

**Issue by GkqqNkKC** [microsoft/playwright#2357]
**Context:**
- Playwright Version: latest `playwright` package
- Operating System: Windows 10
- Node version: 12.14.1

**Describe the bug**

Intellisense does not work for `playwright`. Steps to reproduce:
- `npm i -g playwright`
- create a file in VS Code; my current code is the one below
- hover over `playw`

```
(async () => {
    let lib = require("../libs/node.js")
    let playw = require("playwright")
    process.exit(0)
})()
```

If I hover another module than `playw` I get:

![image](https://user-images.githubusercontent.com/35461917/82889068-df41fd80-9f52-11ea-9832-7462e13214fd.png)

This means if I use `.` operator I get suggestions. If I hover `playw`:

![image](https://user-images.githubusercontent.com/35461917/82889133-f54fbe00-9f52-11ea-8081-3dbeb06a99cf.png)

I'm not sure if this is a bug, or I'm requesting a feature. Feel free to edit the title if necessary. I tested it after restarting VS Code too, same result. I do not use any extension that could influence JS / TS files (only Power Shell scripts)

**Comment by GkqqNkKC** [comment]
Installing it locally works. I tried to require globally installed `request` and `readline` and they work, so something might be related to `playwright`. Feel free to close this issue if the problem doesn't matter. Thx for looking into it

**Comment by pavelfeldman** [comment]
`readline` is a part of Node though. Does anything you install manually with -g work for you? I don't think Playwright is any special with that regard.

**Comment by pavelfeldman** [comment]
Closing for now, please reopen if something new comes up!

**Comment by GkqqNkKC** [comment]
After trying more stuff and testing different packages I came to the conclusion that IntelliSense itself is inconsistent. I got in the first example `playwright` to show it's members, but can't reproduce. It should be safe to assume there's nothing wrong `playwright`, thus the problem becoming out of context (reason I will keep the issue closed). As a rule of thumb, if someone bumps into this issue, a fix may be to fill the code with `;`. As u seen in my example, I don't use them if they're not a must
